vissl/utils/wandb.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+This script contains some helpful functions to handle wandb setup.
+"""
+
+import os
+import logging
+
+from classy_vision.generic.distributed_util import is_primary
+from vissl.utils.checkpoint import get_checkpoint_folder
+from vissl.utils.io import makedir
+from omegaconf import DictConfig
+
+def is_wandb_available():
+    """
+    Check whether wandb is available or not.
```

Comment:
can we add a link here to wandb documentation that open source users can read/access? :)